test/test_nn.py

```diff
@@ -4916,78 +4916,74 @@ def test_shape(N, C, IH, IW, H, W, padding_mode):
             # test CUDA against CPU
             if TEST_CUDA:
                 test_cpu_against_cuda(N, C, H, W, padding_mode)
-
-                # test channels >1024, which doesn't work on cudnn 7102 and further
-                N, C, H, W = 1, 1025, 3, 3
```

Comment:
Why don't we need the >1024 test anymore?

Reply:
I added a test that explicitly disables cudnn below.